src/content/guides/farmlands-card-partner-support.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Farmlands Card Partner Support Guide
+img: /farmlands-pos-integration-cover.jpg
```

Comment:
Do we want to use a different image? This image was for the POS guide

Reply:
We can, there is not currently a specific image that we are supposed to use for this guide so I reused the POS guide image. I will contact the Jesters to see if they have something in mind.

Reply:
@laurencemcknight Sweet - I'd suggest removing this line entirely if they don't have one ready to go.

We default to the image that you see on our core guides if there's no `img` property in the front matter. Chat with them to see which is the most appropriate option 👍 

I'll approve this PR to unblock it

Reply:
Jesters have said to use the default background image here. I will remove the POS integration cover image.